Microsoft.Azure.CosmosRepository/src/Paging/IPage.cs

```diff
@@ -17,6 +17,16 @@ public interface IPage<out T> where T : IItem
         /// </summary>
         int Total { get; }
 
+        /// <summary>
```

Comment:
How do you see these extra properties work when using the existing API? All these properites would not be populated, could we maybe derive another interface from this one with a different name that offers these extra properties? What do you think?